snap/snapcraft.yaml

```diff
@@ -127,6 +127,7 @@ apps:
       - network
       - bluez
       - avahi-observe
+      - process-control
```

Comment:
Why is [process-control](https://snapcraft.io/docs/process-control-interface) interface being added? As issue #8 shows, snappy-debug suggests this interface because there is a seccomp violation on a [sched_setattr](https://man7.org/linux/man-pages/man2/sched_setattr.2.html) system call. Is this affecting the functionality?

See also https://snapcraft.io/docs/debug-snaps#heading--seccomp

Reply:
Yes, it does affect the functionality. Please see the PR description has been updated to include an explanation.

The solution, which involves running the `scmp_sys_resolver` command to solve the Seccomp violation, appears to be less user-friendly compareing to simply adding the process-control plug.

Reply:
> Yes, it does affect the functionality. Please see the PR description has been updated to include an explanation.
> 
> The solution, which involves running the `scmp_sys_resolver` command to solve the Seccomp violation, appears to be less user-friendly compareing to simply adding the process-control plug.

The data you shared shows the same success rate (4/50 timeouts) with and without connecting the interfaces. The delays may be caused by external networking and computing factors. It appears that the tests have been done in batches without randomization. 

I'm not sure what solution are you referring to, involving `scmp_sys_resolver`. The shared document uses that command to resolve the syscall name from its ID.